clients/devtools-entry.js

```diff
@@ -40,6 +40,7 @@ function runLighthouseInWorker(port, url, flags, categoryIDs) {
   // Default to 'info' logging level.
   flags.logLevel = flags.logLevel || 'info';
   flags.channel = 'devtools';
+  global.devtools = true;
```

Comment:
haha, yes, this is terrible :P but at least it works!

I'll play around with it and see what I can do, but I think you're right that `require.resolve()` just fundamentally isn't going to work here...